corehq/apps/hqmedia/templates/hqmedia/audio_translator.html

```diff
@@ -0,0 +1,64 @@
+{% extends "hqmedia/uploader_base.html" %}
+{% load compress %}
+{% load hq_shared_tags %}
+{% load i18n %}
+
+{% block js %}{{ block.super }}
+  {% compress js %}
+    <script src="{% static 'hqwebapp/js/widgets.js' %}"></script>
+  {% endcompress js %}
+{% endblock js %}
+
+{% block page_content %}
+  <p class="lead">{{ current_page.page_name }}</p>
+
+  {% url "app_settings" domain app.get_id as app_settings_url %}
+  {% blocktrans %}
+    <p class="help-block">
+      This page generates files to assist in working with audio translation vendors. It generates a zip file that
+      contains two Excel files:
+    </p>
+    <p class="help-block">
```

Comment:
how about putting this on confluence and linking that page here. I am not sure how many times one would want to see the files available in this download, but would need to log in if they want to.

Reply:
This might be laziness talking, but I don't see the benefit of moving this to confluence. Not sure what you mean about logging in...do you mean that they'd have to log into HQ to see the help text?

Reply:
Yes.

Reply:
got it, going to punt until someone complains